dbfit-java/informix/src/main/java/dbfit/environment/InformixEnvironment.java

```diff
@@ -0,0 +1,331 @@
+package dbfit.environment;
+
+import dbfit.annotations.DatabaseEnvironment;
+import dbfit.api.AbstractDbEnvironment;
+import dbfit.fixture.StatementExecution;
+import dbfit.util.DbParameterAccessor;
+import dbfit.util.DdlStatementExecution;
+import dbfit.util.Direction;
+import dbfit.util.NameNormaliser;
+import dbfit.util.Options;
+import dbfit.util.TypeNormaliserFactory;
+import fit.TypeAdapter;
+
+import java.math.BigDecimal;
+import java.sql.*;
+import java.util.*;
+import java.util.regex.Pattern;
+
+import static dbfit.util.Direction.*;
+
+@DatabaseEnvironment(name="Informix", driver="com.informix.jdbc.IfxDriver")
+public class InformixEnvironment extends AbstractDbEnvironment {
+
+    @Override
+    public void afterConnectionEstablished() throws SQLException {
+        if (currentConnection.getMetaData().supportsTransactions()) {
```

Comment:
I mean - if we have `|set option|autocommit|true|` before `connect` - would `getMetaData().supportsTransactions()` still return `true` or `false`? My concern is that maybe we're overriding the `autocommit=true` with `autocommit=false` which might be confusing.